deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/CodegenConfigTest.java

```diff
@@ -12,14 +12,14 @@ class CodegenConfigTest {
     @Test
     void verifySpaceEncoding() {
         final String resolvedPrefix = CodegenConfig
-                .getBuildTimeSpecPropertyPrefix(Path.of("/home/myuser/luke/my test openapi.json"));
-        assertEquals(resolvedPrefix, String.format(BUILD_TIME_SPEC_PREFIX_FORMAT, "my_test_openapi_json"));
+                .getBuildTimeSpecPropertyPrefix(Path.of("/home/myuser/open-api-stream/luke/my test openapi.json"));
+        assertEquals(String.format(BUILD_TIME_SPEC_PREFIX_FORMAT, "luke_my_test_openapi_json"), resolvedPrefix);
```

Comment:
If you kept the `/home/myuser/luke/my test openapi.json` so the result would be `home_myuser_luke_my_test_openapi_json`? Please keep both tests. You can add a new one verifying the use case where the root path is one of the expected.

Reply:
I restore the previous test and added a new one. 
The idea is that for files generated under $OUTPUTPATH/open-api-stream, the config key will be the relative path (previous it was the file name)
I do not understand how is this related with the readme file.